doc/release-process.md

```diff
@@ -56,35 +56,35 @@ Check out the source code in the following directory hierarchy.
 	git pull
 
 ###Fetch and create inputs: (first time, or when dependency versions change)
- 
+
 	mkdir -p inputs
 	wget -P inputs https://bitcoincore.org/cfields/osslsigncode-Backports-to-1.7.1.patch
 	wget -P inputs http://downloads.sourceforge.net/project/osslsigncode/osslsigncode/osslsigncode-1.7.1.tar.gz
 
- Register and download the Apple SDK: see [OSX readme](README_osx.txt) for details.
- 
+ Register and download the Apple SDK: see [OS X readme](README_osx.txt) for details.
+
  https://developer.apple.com/devcenter/download.action?path=/Developer_Tools/xcode_6.1.1/xcode_6.1.1.dmg
- 
+
  Using a Mac, create a tarball for the 10.9 SDK and copy it to the inputs directory:
- 
+
 	tar -C /Volumes/Xcode/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/ -czf MacOSX10.9.sdk.tar.gz MacOSX10.9.sdk
 
 ###Optional: Seed the Gitian sources cache and offline git repositories
 
-By default, gitian will fetch source files as needed. To cache them ahead of time:
+By default, Gitian will fetch source files as needed. To cache them ahead of time:
 
 	make -C ../bitcoin/depends download SOURCES_PATH=`pwd`/cache/common
 
 Only missing files will be fetched, so this is safe to re-run for each build.
 
-NOTE: Offline builds must use the --url flag to ensure gitian fetches only from local URLs. For example: 
+NOTE: Offline builds must use the --url flag to ensure Gitian fetches only from local URLs. For example:
 ```
 ./bin/bguild --url bitcoin=/path/to/bitcoin,signature=/path/to/sigs {rest of arguments}
```

Comment:
Good pick up @Michagogo I have squashed this change back into the original commit.